addons/map_gestures/stringtable.xml

```diff
@@ -4,116 +4,179 @@
         <Key ID="STR_ACE_map_gestures_moduleSettings_displayName">
             <English>Map Gestures</English>
             <Portuguese>Gestos no mapa</Portuguese>
+            <Polish>Gesty na mapie</Polish>
             <Russian>Жесты на карте</Russian>
         </Key>
         <Key ID="STR_ACE_map_gestures_enabled_displayName">
             <English>Enabled</English>
             <Portuguese>Ativado</Portuguese>
+            <Polish>Aktywne</Polish>
             <Russian>Включено</Russian>
         </Key>
         <Key ID="STR_ACE_map_gestures_maxRange_displayName">
             <English>Map Gesture Max Range</English>
             <Portuguese>Distância para gestos no mapa</Portuguese>
+            <Polish>Maks. zasięg gestów</Polish>
             <Russian>Макс. дистанция жестов на карте</Russian>
         </Key>
         <Key ID="STR_ACE_map_gestures_maxRange_description">
             <English>Max range between players to show the map gesture indicator [default: 7 meters]</English>
             <Portuguese>Distância max. entre os jogadores para mostrar o indicador de gesto no mapa [padrão: 7 metros]</Portuguese>
+            <Polish>Maksymalny zasięg, w obrębie którego gesty będą widoczne dla graczy [domyślnie: 7 metrów]</Polish>
             <Russian>Макс. дистанция между игроками для отображения жестов на карте [по-умолчанию: 7 метров]</Russian>
         </Key>
+        <Key ID="STR_ACE_map_gestures_defaultLeadAlpha_displayName">
```

Comment:
Sorry I didn't know that. I dis this PR back then and updated master today.
I can remove this on my side if needed?